consensus/hotstuff/signature/weighted_signature_aggregator.go

```diff
@@ -129,7 +127,7 @@ func (w *WeightedSignatureAggregator) TrustedAdd(signerID flow.Identifier, sig c
 
 	err := w.aggregator.TrustedAdd(info.index, sig)
 	if err != nil {
-		return currentWeight, fmt.Errorf("Trusted add has failed: %w", err)
+		return w.totalWeight, fmt.Errorf("Trusted add has failed: %w", err)
```

Comment:
currentWeight might get staled, better to read the actual value

Reply:
https://github.com/onflow/flow-go/pull/1347/commits/31aafd421a2dc282bf339de657476c7732b497de